utilities.php

```diff
@@ -25,7 +25,7 @@ function print_listing_li($item_id, $title, $currentPrice, $num_bids, $end_time)
 {
 
   // Fix language of bid vs. bids
-  if ($num_bids = 1) {
+  if ($num_bids <= 1) {

```

Comment:
Thank you so much for pointing that out! I made a quick grammar check, and it should be `0 bids` instead of `0 bid` (English is so weird!). So it might be `$num_bids == 1`.